frontend/packages/knative-plugin/integration-tests/features/managed-kafka connection/kafka-source-connection-to-kafka-connection-instance.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Connection from Kafka Source to Managed Kafka Connection resource
+              As a user, in topology view would like to drag a connection from existing Kafka Source to Managed Kafka Connection (MKC) so that Kafka Source gets updated with bootstrap server url, Security protocol, secrets(creds) so that I don't need to type these
```

Comment:
```suggestion
              As a user, in topology view would like to drag a connection from the existing Kafka Source to Managed Kafka Connection (MKC) so that Kafka Source gets updated with bootstrap server/host url, SASL secrets, TLS enabled so that I don't need to provide those manually
```

Reply:
Updated